### ISSUE TITLE
When showing only Group and Author, the space is missing between the two.

### ISSUE THREAD

**Issue by roberthawdon** [roberthawdon/dfshow#90]

**Comment by roberthawdon** [comment]
This is also causing segfaults occasionally:

```
2019-04-12 10:03:23.193783+0100 show[41007:235657] detected buffer overflow
Process 41007 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
    frame #0: 0x00007fff79df32c6 libsystem_kernel.dylib`__pthread_kill + 10
libsystem_kernel.dylib`__pthread_kill:
->  0x7fff79df32c6 <+10>: jae    0x7fff79df32d0            ; <+20>
    0x7fff79df32c8 <+12>: movq   %rax, %rdi
    0x7fff79df32cb <+15>: jmp    0x7fff79ded457            ; cerror_nocancel
    0x7fff79df32d0 <+20>: retq
Target 0: (show) stopped.
```

**Comment by roberthawdon** [comment]
https://github.com/roberthawdon/dfshow/blob/master/src/showfunctions.c#L669-L672

Introduced in v0.6.1-alpha

**Comment by roberthawdon** [comment]
Fixed in 0.7 branch
